docs/UserGuide.md

```diff
@@ -46,41 +46,43 @@ This guide uses the following features to make it easier for you to navigate aro
 
 <div markdown="block" class="alert alert-warning">
 
-:warning: Boxes with the :warning: icon contain important information about how to keep your Tutor's Pet data safe.
+:warning: Boxes with the :warning: icon contain important information on how to keep your Tutor's Pet data safe.
 
 </div>
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## Quick Start
-This section provides information on how to start using Tutor's Pet.
+This section provides information on how to quickly start using Tutor's Pet.
 
 ### Installing Tutor's Pet
 
 Here are a few steps to get you started on Tutor's Pet:
 
-1. Ensure you have **Java 11** or above installed in your Computer.
+1. Ensure you have **Java 11** or above installed in your computer.
 
-2. Download the latest version of **tutorspet** [here](https://github.com/AY2021S1-CS2103T-T10-4/tp/releases).
+1. Download the latest version of Tutor's Pet [here](https://github.com/AY2021S1-CS2103T-T10-4/tp/releases).
 
-3. Copy the file to the folder you want to use as the home folder for your Tutor's Pet.
+1. Copy the downloaded Tutor's Pet to a folder of your choosing. This shall be known as the home folder of your Tutor's Pet.
 
-4. Double-click the file to start the application. An application similar to the one below should appear in a few
+1. Double-click the file to start the application. An application similar to the one below should appear in a few
    seconds.<br>
 
    ![Ui](images/Ui.png)
 
 ### Using Tutor's Pet
 
-This section offers an overview of Tutor's Pet layout.
+This section offers an overview of the layout in Tutor's Pet.
 
-There are three main areas in Tutor's Pet:
+There are four main areas in Tutor's Pet:
```

Comment:
Is there are particular reason as to why we want to split the result display vs command box?

Reply:
It looks more intuitive to split the boxes and the screenshot got the different labels also.